generated/csharp/GitHub/Middleware/VersionHandler.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.Kiota.Abstractions;
+
+namespace GitHub.Octokit.Client.Middleware;
+
+class VersionHandler : DelegatingHandler
+{
+    private const string ApiVersionHeaderKey = "X-GitHub-Api-Version";
+
+    private readonly string _apiVersion;
+
+    public VersionHandler(string apiVersion = "2022-11-28")
```

Comment:
What are your thoughts about making the function in VersionOptions.cs public to avoid defining this twice?

Reply:
Heh, I actually already got rid of the other one, I just never cleaned up the orphaned code. Good catch!